pom.xml

```diff
@@ -315,6 +332,38 @@
       <version>${jsoup.version}</version>
     </dependency>
 
+    <!-- Fluency logback dependencies -->
```

Comment:
Doh. Thanks! That's a good catch. I moved them into the default since the conditionals are in place and the jars aren't too large. The fluency profile was a carry over from Cantaloupe's config. I'll delete the profile.